tests/validation/tests/v3_api/test_deploy_clusters.py

```diff
@@ -0,0 +1,53 @@
+from .common import *  # NOQA
+from .test_rke_cluster_provisioning import create_and_validate_custom_host
+
+if_not_auto_deploy_rke = pytest.mark.skipif(
+    ast.literal_eval(
+        os.environ.get(
+            'RANCHER_TEST_DEPLOY_RKE', "False")) is False,
+    reason='auto deploy rke tests are skipped')
+
+
+@if_not_auto_deploy_rke
+def test_deploy_rke():
+    print("Deploying RKE Clusters")
+
+    # TODO: Change kdm if env variable supplied (in module fixture)
+
+    rancher_version = get_setting_value_by_name('server-version')
+    if str(rancher_version).startswith('v2.2'):
+        k8s_v = get_setting_value_by_name('k8s-version-to-images')
+        default_k8s_versions = json.loads(k8s_v).keys()
+    else:
+        k8s_v = get_setting_value_by_name('k8s-versions-current')
+        default_k8s_versions = k8s_v.split(",")
+
+    # Create clusters
+    env_details = "env.CLUSTER_NAMES='"
+    for k8s_version in default_k8s_versions:
+        if env_details != "env.CLUSTER_NAMES='":
+            env_details += ","
+        print("Deploying RKE Cluster using kubernetes version {}".format(
+            k8s_version))
+        node_roles = [["controlplane"], ["etcd"],
+                      ["worker"], ["worker"], ["worker"]]
+        cluster, aws_nodes = create_and_validate_custom_host(
+            node_roles, random_cluster_name=True, version=k8s_version)
+        env_details += cluster.name
+        print("Successfully deployed {} with kubernetes version {}".format(
+            cluster.name, k8s_version))
+    env_details += "'"
+    print("Env details: {}".format(env_details))
+    create_config_file(env_details)
```

Comment:
Why do we need to create the config file here?

Reply:
So that the new environment variable is available (`env.CLUSTER_NAMES`) and the Jenkinsfile can load it in: 
```
// copy file containing CLUSTER_NAMES and load into environment variables
sh "docker cp ${clusterSetupContainer}:${rootPath}${testsDir}${rancherConfig} ."
load rancherConfig
```